capstone/src/main/webapp/explore-loader.js

```diff
@@ -60,9 +61,12 @@ function getLabelQueryString () {
   return queryString.slice(0, -1); // Takes out the last comma. 
 }
 
-function getJoinButton(clubName) {
-  var joinButton = '<button name="join" value="'
-    + clubName
-    + '" formmethod="POST">Join Club</button>';
-  return joinButton;
-}
\ No newline at end of file
+function sendJoinedAlert(interestedOrJoin) {
+  var alertMessage = 'You have successfully ';
+  if (interestedOrJoin == 'interested') {
+    alertMessage += 'expressed interest in this club! You can view your interested club list on your profile page.'
```

Comment:
I guess this works without the semicolons, but might be better to add them for consistency

Reply:
Oh, oops haha must have forgot to add them. Thanks Megan!